CHANGELOG.md

```diff
@@ -5,8 +5,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Added the `advisories` check and configuration section for checking crates against an advisory database to detect security vulnerabilities, unmaintained crates, and crates with security notices
+- A warning will now be emitted if a crate that isn't in the graph is specified in `[bans.skip-tree]`
+
 ### Fixed
 - [PR#58](https://github.com/EmbarkStudios/cargo-deny/pull/58) Fixed [#55](https://github.com/EmbarkStudios/cargo-deny/issues/55) to handle license requirements for GPL, AGPL, LGPL, and GFDL better. Thank for reporting [@pikajude](https://github.com/pikajude)!
+- [PR#62](https://github.com/EmbarkStudios/cargo-deny/pull/62) Fixed [#56](https://github.com/EmbarkStudios/cargo-deny/issues/56), the `[metadata]` section in `Cargo.lock` is now gone in nightly to improve merging, the previous reporting mechanism that required this section has been reworked.
+
+### Changed
+- Renamed the `<which>` options for `cargo deny check <which>` from `ban` => `bans` and `license` => `licenses`
```

Comment:
This is a pretty big breaking change if someone is running it in CI, but not sure if we can highlight it more. And we are not at 1.0 yet

Reply:
Yah, this just made it more consistent with the configuration names and everything else. But I can instead make it have both the old and new and instead mark them as deprecated.

Reply:
That would be cleaner, and doesn't hurt anything right now. And good to get into the practice of backwards compatibility / deprecation.